src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,18 @@
-export const TodoList = () => {};
+import { Todo } from '../../types/Todo';
```

Comment:
Consider using more descriptive name for `Todo` type. For example, `TodoItem` or `TodoTask`. 📝